doc/modules/model_persistence.rst

```diff
@@ -51,10 +51,9 @@ with::
 
 .. note::
 
-   joblib.dump returns a list of filenames. Each individual numpy array
-   contained in the ``clf`` object is serialized as a separate file on the
-   filesystem. All files are required in the same folder when reloading the
-   model with joblib.load.
+   ``joblib.dump`` and ``joblib.load`` functions also accept file-like object
+   instead of filenames, more information on persistence in Joblib is
+   available `here <https://pythonhosted.org/joblib/persistence.html>`.
```

Comment:
this link is malformed, you need an `_` at the end (e.g. \`here \<https://pythonhosted.org/joblib/persistence.html>`_)